ESP8266/ESP8266.cpp

```diff
@@ -547,6 +560,14 @@ void ESP8266::_connect_error_handler()
     }
 }
 
+void ESP8266::_oob_socket_close_error()
+{
+    if (_parser.recv("ERROR\n")) {
```

Comment:
If the OOB is "UNLINK ERROR", but triggers with "UNLINK", should you be including also the space somewhere?